kafka-connect-transform-kryptonite-gcp/src/main/java/com/github/hpgrahsl/kafka/connect/transforms/kryptonite/CipherField.java

```diff
@@ -105,6 +107,8 @@
   private static final String KEY_SOURCE_DEFAULT = "CONFIG";
   private static final String CIPHER_DATA_KEYS_DEFAULT = "[]";
   private static final String KMS_KEY_NAME_DEFAULT = null;
+  private static final int KRYO_OUTPUT_BUFFER_SIZE_DEFAULT = 32;
```

Comment:
This is same as the default size of ByteArrayOutputStream
https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/19fb8f93c59dfd791f62d41f332db9e306bc1422/src/java.base/share/classes/java/io/ByteArrayOutputStream.java#L64